test/parallel/test-tls-net-connect-prefer-path.js

```diff
@@ -28,8 +29,8 @@ function mkServer(lib, tcp, cb) {
   const args = [handler];
   if (lib === tls) {
     args.unshift({
-      cert: fs.readFileSync(`${common.fixturesDir}/test_cert.pem`),
-      key: fs.readFileSync(`${common.fixturesDir}/test_key.pem`)
+      cert: fs.readFileSync(fixtures.path('test_cert.pem')),
+      key: fs.readFileSync(fixtures.path('test_key.pem'))
```

Comment:
The `fixtures` module has a helper that make this even simpler! Check out [`fixtures.readSync()`](https://github.com/nodejs/node/tree/master/test/common#fixturesreadsyncargs-enc).

That should let you do `key: fixtures.readSync('test_key.pem')` instead.